images/nat-image/installAnsible.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
why are you installing ansible? Packer can automatically use ansible as a provisioner while creating the image